Update overview.md (#5638)
MicrosoftDocs/windows-uwp: hub/apps/develop/smart-app-control/overview.md

- If **Evaluation** is selected, Smart App Control is running in evaluation mode.
- If **Off** is selected, Smart App Control is not running on this device.

### What makes somebody a good candidate for Smart App Control?

> Essentially, we're looking to see if Smart App Control is a good fit for your device or if it is going to get in your way too often. **In most cases Smart App Control will automatically turn on** to protect againt untrusted or malicious apps. However, there are some legitimate tasks that corporate users, developers, or others may do regularly that may not be a great experience with Smart App Control running. **If we detect that you're one of those users, we'll automatically turn Smart App Control off** so you can work with fewer interruptions.

Reviewer: Some minor edits based on our style recommendations for Learn.
```suggestion

Essentially, we're looking to see if Smart App Control is a good fit for your device or if it's going to get in your way too often. **In most cases Smart App Control automatically turns on** to protect against untrusted or malicious apps. However, there are some legitimate tasks that corporate users, developers, or others do regularly that might not be a great experience with Smart App Control running. **If we detect that you're one of those users, we automatically turn Smart App Control off** so you can work with fewer interruptions.
```
